dotCMS/src/integration-test/java/com/dotmarketing/util/HashBuilderTest.java

```diff
@@ -0,0 +1,33 @@
+package com.dotmarketing.util;
+
+import com.liferay.util.Encryptor;
+import com.liferay.util.HashBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.charset.StandardCharsets;
+
+public class HashBuilderTest {
+
+    @Test
+    public void Test_BuildUnixHash () throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6220272)